app.py

```diff
@@ -187,7 +188,8 @@ def prepare_body_headers_with_data(request):
                     "roleInformation": AZURE_OPENAI_SYSTEM_MESSAGE,
                     "embeddingEndpoint": AZURE_OPENAI_EMBEDDING_ENDPOINT,
                     "embeddingKey": AZURE_OPENAI_EMBEDDING_KEY,
-                    "filter": filter
+                    "filter": filter,
+                    "threshold": float(AZURE_SEARCH_THRESHOLD) if AZURE_SEARCH_THRESHOLD else 0.3
```

Comment:
I'm curious what situation the if/else accounts for. Empty string?

Reply:
Yeah, empty string. In case the setting gets deleted or reset to empty, otherwise the float case will fail